### DISCUSSION TITLE
对于需要git submodule的库怎么做xmake包？

### DISCUSSION THREAD

**Discussion by happyxianyu** [xmake-io/xmake#2376] [Q&A]
如题

**Comment by waruqi** [comment]
正常做 参考 xmake-repo

**Reply by happyxianyu** [reply]
是直接在下载下来的仓库里执行git submodule命令吗？我在xmake-repo里全代码搜索了一下，没看到哪个package用了git submodule命令，是有更好的办法吗？比如有内置模块提供的功能，如果是的话，请问有没有合适的例子可以参考？

**Reply by waruqi** [reply]
带 git url 就行了，xmake-repo 里面随便找个有 git url 的包 https://github.com/xmake-io/xmake-repo/blob/42bf1fa0b15d271c477bfc17315d0a778a256edd/packages/x/x264/xmake.lua#L6

**Reply by happyxianyu** [reply]
好的，谢谢，我看看
